templates_test/main_test/mssql_main.tpl

```diff
@@ -0,0 +1,127 @@
+type mssqlTester struct {
+	dbConn     *sql.DB
+	dbName     string
+	host       string
+	user       string
+	pass       string
+	sslmode    string
+	port       int
+	testDBName string
+}
+
+func init() {
+	dbMain = &mssqlTester{}
+}
+
+func (m *mssqlTester) setup() error {
+	var err error
+	m.dbName = viper.GetString("mssql.dbname")
+	m.host = viper.GetString("mssql.host")
+	m.user = viper.GetString("mssql.user")
+	m.pass = viper.GetString("mssql.pass")
+	m.port = viper.GetInt("mssql.port")
+	m.sslmode = viper.GetString("mssql.sslmode")
+	// Create a randomized db name.
+	m.testDBName = randomize.StableDBName(m.dbName)
+
+	if err = m.dropTestDB(); err != nil {
+		return err
+	}
+	if err = m.createTestDB(); err != nil {
+		return err
+	}
+
+	createCmd := exec.Command("sqlcmd", "-S", m.host, "-U", m.user, "-P", m.pass, "-d", m.testDBName)
+
+	f, err := os.Open("tables_schema.sql")
```

Comment:
I couldn't recall if the Go test will always run the test local to the package.

So I tested by creating a package called testdir that read it's own test file for the test. It passed when run from my homedir as `go test testdir`. This is just a comment for anyone else who might look at this as suspiciously as I did.